### DISCUSSION TITLE
Allow different reduced scoring percentages for different sets

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork3#40] [Ideas]
webwork2 has a single reduced scoring percentage for the entire course.  I'd like to be able to set different reduced scoring amounts for different sets (e.g. Assignment 1 is worth 50% during the reduced scoring period, but Assignment 2 is worth 70% during the reduced scoring period).

This could be taken even further in allowing multiple reduced scoring periods for the same set.  e.g. for the first two days students can receive 70% credit, and for the next two days they can receive 50% credit.

**Comment by JosephMPedersen** [comment]
WebAssign has the ability to set the late penalty based on HOW late the assignment was turned in. Problems answered correctly before the due date receive 100% of the points, but problems answered correctly N-days late receive (100 - N*L)% of the points.  Would that be very hard to do in WeBWorK?  I'd like to start using WeBWorK, and that would be a very important feature for me, since that is how our department's late policy works (all courses have 5% per day late penalty).  I'll try to code that functionality myself if possible, although my only experience in Perl has been using it to write questions in WebAssign and WeBWorK.
